src/videojs-media-sources.js

```diff
@@ -191,9 +191,18 @@ addTextTrackData = function (sourceHandler, captionArray, metadataArray) {
       // MediaSource
       this.sourceBuffers = [];
 
+      // Re-emit MediaSource events on the polyfill
+      [
+        'sourceopen',
+        'sourceclose',
+        'sourceended'
+      ].forEach(function(eventName) {
+        this.mediaSource_.addEventListener(eventName, this.trigger.bind(this));
```

Comment:
not necessary to change or anything, but if you have a reference to videojs, you could do `videojs.on(this.mediaSource_, ['sourceopen', 'sourceclose', 'sourceended'], this.trigger.bind(this)))`